okhttp/src/main/java/io/grpc/okhttp/OkHttpChannelBuilder.java

```diff
@@ -463,8 +487,8 @@ SSLSocketFactory createSocketFactory() {
     private final boolean usingSharedExecutor;
     private final boolean usingSharedScheduler;
     private final TransportTracer.Factory transportTracerFactory;
-    @Nullable
-    private final SSLSocketFactory socketFactory;
+    @Nullable private final SocketFactory socketFactory;
```

Comment:
Remove `@Nullable`, it's just like a random NPE from any library if ` SocketFactory.getDefault()` were null.

Reply:
Done